app/react/components/LoggedInNav.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default (props) => {
+  return (
+    <nav className="mdl-navigation">
+      {props.auth.get('role') === 'admin' ? <Link to="admin-dashboard" className="mdl-navigation__link">All Cases</Link> : ''}
+      {props.auth.get('role') === 'admin' ? <Link to="advocates" className="mdl-navigation__link">Advocates</Link> : ''}
+      {props.auth.get('role') === 'admin' ? <Link to="volunteers" className="mdl-navigation__link">Volunteers</Link> : ''}
```

Comment:
There are a lot of ternaries here for all the same thing. Is this something to refactor, do we think? This may still be better than having two different components, for example--just a thought.